component/elasticsearch.libsonnet

```diff
@@ -0,0 +1,127 @@
+// main template for openshift4-lokistack
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local resourceLocker = import 'lib/resource-locker.libjsonnet';
+
+// The hiera parameters for the component
+local inv = kap.inventory();
+local elasticsearch = inv.parameters.openshift4_logging.components.elasticsearch;
+
+
+local machineconfig_journald = [
+  kube._Object('machineconfiguration.openshift.io/v1', 'MachineConfig', '41-' + role + '-journald') {
```

Comment:
I'd try very hard to keep the names of the machineconfiguration objects the same, since I'm not sure whether the name of a machineconfig is taken into consideration by the machine-config-operator when rendering the final machine configs for the machine config pools. If it is, this change will cause rolling node restarts, which I think we should try to avoid to allow us to roll out the new component version during business hours.